### ISSUE TITLE
Deep dermatophytosis

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#4845]
**Preferred term label:**
Deep dermatophytosis

**Synonyms**

**Definition (free text, please give PubMed ID)**
see https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4084693/

**Parent term (use hpo.jax.org/app)**
probably https://hpo.jax.org/app/browse/term/HP:0032162

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
see https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4084693/

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Many of the terms needed for this disease (https://omim.org/entry/212050) need to be added actually.
